src/Realizer.js

```diff
@@ -8,9 +8,38 @@ class Realizer
 		this.isFullyUpdated = false;
 		this.moveQueue = [];
 		
+		this.activeCell = -1;
```

Comment:
I would recommend defaulting this to `undefined` instead. I realize `-1` is supposed to be an invalid value. Since it is a number, if we have another bug in the code that defines `board.contents[-1]` - that would not cause problems here when indeed it should.